package.json

```
@@ -3,18 +3,16 @@
   "version": "0.4.0",
   "scripts": {
     "build": "rollup -c",
-    "dev": "rollup -c --config-enable-developer-mode",
-    "lint": "eslint . && web-ext lint",
+    "dev": "npm-run-all --parallel dev:extension dev:webext",
+    "lint": "npm run build && eslint . && web-ext lint",
     "package": "npm run build && web-ext build --overwrite-dest --ignore-files \"./tests/**/*\" --ignore-files \"./src/**/*\" --ignore-files \"./rollup.config.js\"",
     "start": "npm run build && web-ext run",
-    "start-dev": "npm run dev && web-ext run",
-    "test-integration": "npm run package && mv web-ext-artifacts/*.zip web-ext-artifacts/study.xpi && mocha --timeout 30000 \"./tests/integration/*.js\"",
-    "watch": "npm-run-all --parallel watch-raw watch-bundled",
-    "watch-raw": "npm run dev -- -w",
-    "watch-bundled": "web-ext run --watch-file dist/background.js"
+    "test:integration": "npm run package && mv web-ext-artifacts/*.zip web-ext-artifacts/study.xpi && mocha --timeout 30000 \"./tests/integration/*.js\"",
+    "dev:extension": "rollup -c --config-enable-developer-mode -w",
+    "dev:webext": "web-ext run --watch-file dist/background.js"
   },
   "devDependencies": {
-    "@mozilla/rally": "^0.6.0",
+    "@mozilla/rally": "github:rhelmer/rally-sdk#ca34d37a4b4000228dc3649f0861394a37ea54e3",
```

```suggestion
    "@mozilla/rally": "github:mozilla-rally/rally-sdk#b3bb288106f3fe66619fae92c9608e6fca6f5e70",
```

The changes we needed landed in the official repo, my fork was only for testing.